### PR TITLE
add count

### PR DESCRIPTION
**Pull request by iiiiii1wepfj**

### CODE REVIEW

**Comment by iiiiii1wepfj** [comment]
done

**Review by AndrielFR — APPROVED** [review]

File: eduu/plugins/youtube.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ async def ytdlcmd(c: Client, m: Message, strings):
 
     if m.reply_to_message and m.reply_to_message.text:
         url = m.reply_to_message.text
-    elif len(m.command) > 1:
+    elif len(m.text.split()) > 1:
```
Comment (AndrielFR) [inline eduu/plugins/youtube.py:37]: Ok, for a pull request for a change in `welcome`, that doesn't make sense, but ok.

Comment (iiiiii1wepfj) [reply eduu/plugins/youtube.py:37]: @AndrielFR if i am here why dont fix this too?

Comment (AndrielFR) [reply eduu/plugins/youtube.py:37]: Idk.

Comment (alissonlauffer) [reply eduu/plugins/youtube.py:37]: What's wrong here? It is working fine here.

Comment (iiiiii1wepfj) [reply eduu/plugins/youtube.py:37]: @alissonlauffer no, check amano chat

File: eduu/plugins/youtube.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ async def ytdlcmd(c: Client, m: Message, strings):
 
     if m.reply_to_message and m.reply_to_message.text:
         url = m.reply_to_message.text
-    elif len(m.text.split()) > 1:
+    elif len(m.command) > 1:
```
Comment (AndrielFR) [inline eduu/plugins/youtube.py]: Idk what you're getting at.

**Comment by iiiiii1wepfj** [comment]
now this is good?

**Pull request merged**